framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -342,7 +342,7 @@
     public final boolean ignoreUninferredTypeArguments;
 
     /** The Object.getClass method. */
-    protected final ExecutableElement objectGetClass;
+    public final ExecutableElement objectGetClass;
```

Comment:
Is the only use here https://github.com/typetools/checker-framework-inference/pull/97/files ?
That class is a sub-class of this class, so `protected` should be enough, shouldn't it?